### PR TITLE
feat: add a scriv-based changelog

### PR DESCRIPTION
**Pull request by regisb**
This is to address https://github.com/overhangio/tutor/issues/746

### CODE REVIEW

**Comment by regisb** [comment]
@overhangio/tutor-maintainers could you please review this PR? If you're happy with the proposed changes, please open a similar PR in the repos you maintain. Thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+<a id='changelog-15.0.2'></a>
+## v15.0.2 (2023-03-20)
+
+- [Improvement] Add a changelog to this plugin. (by @regisb)
+
+- Pouac.
```
Comment (kdmccormick) [inline CHANGELOG.md]: Pouac?

**Review by kdmccormick — APPROVED** [review]
Looks good.

I will take care of this for the cookiecutter repo after the conference: https://github.com/overhangio/cookiecutter-tutor-plugin/issues/20

**Review by arbrandes — APPROVED** [review]
Looks good! 👍🏼

**Comment by arbrandes** [comment]
For tutor-mfe: https://github.com/overhangio/tutor-mfe/issues/118

**Comment by ghassanmas** [comment]
I wonder would it be okay to change the Readme file extension from `rst` to `md`? Would that have any consequences long or short term

File: CHANGELOG.md (hunk 1 above)
Comment (regisb) [reply CHANGELOG.md]: No. This totally never happened.

**Comment by regisb** [comment]
> I wonder would it be okay to change the Readme file extension from rst to md? Would that have any consequences long or short term

There are no serious consequences that I can think of. I picked rst as a README format early on, because in my experience this makes it much easier to convert a long readme to a full sphinx-compatible docs site. I'm not sure I would agree on an rst -> md conversion, but we can certainly discuss it.

**Pull request merged**